src/scss/grommet-core/_objects.video.scss

```diff
@@ -37,15 +41,20 @@
 
   &__summary {
     position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+    top: 0%;
+    width: 100%;
+    height: 100%;
     display: flex;
     align-items: center;
-    text-align: left;
     padding: $inuit-base-spacing-unit;
   }
 
+  &--video-header {
+    .video__summary {
+      padding: 0;
```

Comment:
It's the default behavior for media-query(lap-and-up):
```
  &__summary {
    position: absolute;
    top: 0;
    width: 100%;
    height: 100%;
    display: flex;
    align-items: center;
    padding: $inuit-base-spacing-unit;
  }
```

When there's a header component being passed in, we don't want extra padding on the header.